app/views/layouts/white_theme.html.erb

```diff
@@ -36,6 +36,7 @@
 
   <header id="site_header" data-turbolinks-permanent>
     <%= render partial: 'shared/header' %>
+    <%= render partial: 'shared/admin' %>
```

Comment:
Oh, we probably don't want to keep this (or the partial) now that we have a side nav.